web/components/Navbar.tsx

```diff
@@ -23,8 +23,10 @@ const useStyles = makeStyles({
   },
 });
 
-type NavbarProps = {};
-export const Navbar: React.FC<NavbarProps> = () => {
+type NavbarProps = {
+  user: any;
+};
+export const Navbar: React.FC<NavbarProps> = (props) => {
```

Comment:
```suggestion
export const Navbar: React.FC<NavbarProps> = ({ user }) => {
```
we can destructure our props so we don't need to say `props.something` everytime we want to use our props